### ISSUE TITLE
some files with variables are not being parsed

### ISSUE THREAD

**Issue by atfzl** [atfzl/eslint-plugin-css-modules#11]
```scss
:root {
  --color: red;
}
```

**Comment by atfzl** [comment]
the latest version of gonzales-pe is not able to parse this but some other version of gonzales (primitive) is able to parse this.

Checked on http://tonyganch.com/gonzales-pe/

We should first parse the file using the latest version, if it fails then try with the primitive version.

**Comment by atfzl** [comment]
Fixed in 2.6.0

@gdub22 the parsing which was failing for some cases should be fixed now.

**Comment by gdub22** [comment]
@atfzl it's working well. However, I now see a bunch of logs when linting, which look like they are probably coming from the underlying parser? They look like this:
```
Node {
  type: 'character',
  content: 'px',
  syntax: 'css',
  start: { line: 77, column: 13 },
  end: { line: 77, column: 13 } }
```

**Comment by atfzl** [comment]
There was a console.log statement left in gonzales-primitives.js.
Thank you for reporting all these bugs.
Fixed in 2.6.1
